### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by kikiondo**
# Description:
This branch has two different features.

* First: Generate CSV Files

* Second: Generate PDF Files

### CODE REVIEW

**Comment by fernandoamz** [comment]
commits must be in english

**Review by 8geonirt — APPROVED** [review]
LGTM!!! I agree with @fernandoamz, try to use commits in english next time 🤓 
Nice job

**Review by diegosaul402 — APPROVED** [review]
Excellent! :shipit:

File: Gemfile [hunk 1]
```diff
@@ -20,6 +20,10 @@ gem 'devise'
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
+
+gem 'wicked_pdf'
```
Comment (archisss) [inline Gemfile:24]: nice GEM

**Review by archisss — APPROVED** [review]

**Review by fernandoamz — APPROVED** [review]
awesome!

**Review by archisss — APPROVED** [review]
👨‍💻

**Pull request merged**